x-pack/packages/kbn-ai-assistant/src/chat/chat_body.tsx

```diff
@@ -226,9 +226,36 @@ export function ChatBody({
   });
 
   const handleCopyConversation = () => {
+    const deserializeMessage = (message: Message): Message => {
+      const copiedMessage = cloneDeep(message);
+
+      const { function_call: functionCall, content, data } = copiedMessage.message;
+
+      if (
+        copiedMessage.message.function_call?.arguments &&
```

Comment:
`copiedMessage.message.function_call` is the same thing with `functionCall` right ?

Would something like `if ( functionCall?.arguments && typeof functionCall.arguments === 'string ) {` simpler ? 

Same below as well to make it easier to follow along.

Reply:
Hi @efegurkan 
Thank you for the review. I'm unable to completely replace this line with `if ( functionCall?.arguments && typeof functionCall.arguments === 'string ) { ` because function_call might be undefined. That disallows me to set the deserialized arguments back to `copiedMessage.message.function_call.arguments`. Therefore, I removed the de-structuring for consistency.